packages/vision/src/index.js

```diff
@@ -575,9 +615,93 @@ Vision.prototype.detect = function(images, options, callback) {
     });
   });
 };
+
 // jscs:enable maximumLineLength
 
 /**
+ * Detect the crops within an image.
```

Comment:
Given the absence of official documentation, I'm honestly not sure how to do any better. `@resource` link fixed, though 👍 (I think, hard to know for sure since the docs aren't public)